remix/app/repositories/user.server.test.ts

```diff
@@ -0,0 +1,55 @@
+import type { User } from "@prisma/client";
+import { prismaMock } from "../tests/setup-test-env";
+import UserRespository from "../repositories/user.server"
+
+describe("Test user model", () => {
+    test("Create User", async () => {
+        const user: User = {
+            name: "Beckham",
```

Comment:
Creating all these attributes by hand will get tiring and repetitive. It is best to use factories for text pictures: https://github.com/nimblehq/js-framework-benchmark/blob/develop/nextjs-12/test/factories/user.factory.ts